stubs/ldap3/ldap3/utils/tls_backport.pyi

```diff
@@ -0,0 +1,3 @@
+class CertificateError(ValueError): ...
+
+def match_hostname(cert, hostname): ...
```

Comment:
Alternatively, we could make `types-ldap3` depend on `types-backports.ssl-match-hostname`, and re-export from there.

Reply:
As `ldap3` doesn't depend on the backports package, it will use its own version by default. I think we should do the same in the stubs.